object_test.go

```diff
@@ -113,17 +119,40 @@ func TestObjectSignalHandling(t *testing.T) {
 		}()
 
 		// desired signals
-		bus.Emit(path, iface+".Heartbeat", uint32(1))
-		bus.Emit(path, iface+".Heartbeat", uint32(2))
+		err := bus.Emit(path, iface+".Heartbeat", uint32(1))
+		if err != nil {
+			t.Error(err)
+		}
+		err = bus.Emit(path, iface+".Heartbeat", uint32(2))
+		if err != nil {
+			t.Error(err)
+		}
+
 		// undesired signals
-		bus.Emit(otherPath, iface+".Heartbeat", uint32(3))
-		bus.Emit(otherPath, otherIface+".Heartbeat", uint32(4))
-		bus.Emit(path, iface+".Updated", false)
+		err = bus.Emit(otherPath, iface+".Heartbeat", uint32(3))
+		if err == nil {
+			t.Error("expected error, got nil")
+		}
+		err = bus.Emit(otherPath, otherIface+".Heartbeat", uint32(4))
+		if err == nil {
+			t.Error("expected error, got nil")
+		}
```

Comment:
Please check if I made the right choice here.

Reply:
OK, since this is sort of out of scope of "let's add linters", I created a separate PR which fixes this issue: https://github.com/godbus/dbus/pull/313

Will rebase this one on top once #313 is merged.

Reply:
Rebased but moved this one to draft until #313 is merged.

Reply:
PTAL @guelfey ^^^ (and #313).